### PR TITLE
Create help.md

### PR DESCRIPTION
**Pull request by logankilpatrick**

### CODE REVIEW

File: about/help.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+## New to Julia?
```
Comment (tlienart) [inline about/help.md:1]: Given that you have a bunch of lists here, and that using `@@tight-list` is annoying, you could just put this at the top:

```
~~~
<style>
  .container li > p {margin-bottom: 0;}
</style>
~~~
```

The reason for this is that the Julia Markdown to HTML processor adds `<p></p>` in  list items, these inherit properties for the whole document and it leads to unsightly big vertical spaces between list items.

File: about/help.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+## Do you have a Julia question?
+ - First check the [Julia FAQs](https://docs.julialang.org/en/v1/manual/faq/) section of the docs which includes answers to many common, general Julia questions.
+ - The next best place to go to ask a question is the offical [Julia Discourse site](https://discourse.julialang.org).
+ - Stack Overflow has a growing number of [Julia questions and answers](https://stackoverflow.com/questions/tagged/julia).
+ - For realtime help from a person, you should check out our [official chat channels](/community/#chat).
+ - If you think there may be a bug in core Julia, search the [open issues on the core Julia repo](https://github.com/JuliaLang/julia/issues).
+ - If you think you may have found a security vulnerability in Julia, please read the [instructions for reporting security issues](https://github.com/JuliaLang/julia/security/policy).
+ - If you think you have found a problem with this website, please see the [issues section for the website repository](https://github.com/JuliaLang/www.julialang.org/issues).
+
+## Are you looking for a paticular Julia Package?
```
Comment (tlienart) [inline about/help.md:18]: particular

File: about/help.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+ - Check out the [packages page](/packages/) to browse the complete list of all registered Julia Packages.
+
+## Want to contribute?
+ - To report a bug in core Julia, use the [Julia issue tracker on GitHub](https://github.com/JuliaLang/julia/issues).
+ - To report a problem with this web site, see the [issues section for the website repository](https://github.com/JuliaLang/www.julialang.org).
```
Comment (tlienart) [inline about/help.md:23]: website

Also, nitpicking, the style of the question is different, the previous one was "Are you looking..." and here is the more casual "Want to contribute", just use one of the two maybe?

**Comment by logankilpatrick** [comment]
@tlienart updated all the things you suggested and yes, nothing points to this now but I will add it to the footer.

**Pull request merged**